ports/stm32/boards/NUCLEO_H563ZI/mpconfigboard.h

```diff
@@ -0,0 +1,80 @@
+#define MICROPY_HW_BOARD_NAME               "NUCLEO_H563ZI"
+#define MICROPY_HW_MCU_NAME                 "STM32H563ZI"
+
+#define MICROPY_PY_PYB_LEGACY               (0)
+#define MICROPY_HW_ENABLE_INTERNAL_FLASH_STORAGE (1)
+#define MICROPY_HW_ENABLE_RTC               (1)
+#define MICROPY_HW_ENABLE_RNG               (1)
+#define MICROPY_HW_ENABLE_ADC               (1)
+#define MICROPY_HW_ENABLE_DAC               (0) // TODO: requires DMA, which is not yet ready
+#define MICROPY_HW_ENABLE_USB               (1)
+#define MICROPY_HW_HAS_SWITCH               (1)
+#define MICROPY_HW_HAS_FLASH                (1)
+
+// The board has a 8MHz oscillator, the following gives 250MHz CPU speed
+#define MICROPY_HW_CLK_USE_BYPASS           (1)
+#define MICROPY_HW_CLK_PLLM                 (2)
+#define MICROPY_HW_CLK_PLLN                 (125)
+#define MICROPY_HW_CLK_PLLP                 (2)
+#define MICROPY_HW_CLK_PLLQ                 (2)
+#define MICROPY_HW_CLK_PLLR                 (2)
+#define MICROPY_HW_CLK_PLLVCI_LL            (LL_RCC_PLLINPUTRANGE_4_8)
+#define MICROPY_HW_CLK_PLLVCO_LL            (LL_RCC_PLLVCORANGE_WIDE)
+#define MICROPY_HW_CLK_PLLFRAC              (0)
+
+// PLL3 with Q output at 48MHz for USB
```

Comment:
Please enable `MICROPY_HW_CLK_USE_PLL3_FOR_USB` for this to work (see 67b1c81f69d6cd1c5ad9c35bb68f4490e94865e5)

Reply:
After carefully checking the settings I found the value of PLL3 are not correct. The USB function only worked, because the internal 48 MHz oscillator HSI 48) was used. This is actually quite convenient as the PLL is not required and the external crystal frequency doesn't matter. Actually a nice STM32H5 feature, as it tunes the USB clock based on the USB SOF (start of frame) signal.

Should we keep on using HSI 48 and remove the unnecessary PLL3 #defines? Or do you see an advantage in running the USB controller from a proper 48 MHz clock?

Reply:
Fix incorrect PLL definition. Leaving PLL currently disabled, until I can properly test it.

Reply:
OK, we can leave this disabled to use HSI for USB.